.github/workflows/ci.yml

```diff
@@ -22,6 +22,10 @@ jobs:
         - { name: Static, flags: -DBUILD_SHARED_LIBS=FALSE }
 
     steps:
+    - name: Install Window Dependencies
+      if: runner.os == 'Windows'
+      run: choco install ninja
+
```

Comment:
Just realized, why are we installing ninja? We aren't using it... 🤔

Reply:
The MinGW jobs uses it

Reply:
Oh I didn't see that sneaking -G 😁